docs/using/specifics.md

```diff
@@ -39,51 +39,10 @@ ipython profile create
 
 ### Build an Image with a Different Version of Spark
 
-You can build a `pyspark-notebook` image with a different `Spark` version by overriding the default value of the following arguments at build time.
+You can build a `pyspark-notebook` image with a different `Spark` version by adjusting
+the ['setup_spark.py'](https://github.com/jupyter/docker-stacks/blob/main/images/pyspark-notebook/setup_spark.py) script.
 `all-spark-notebook` is inherited from `pyspark-notebook`, so you have to first build `pyspark-notebook` and then `all-spark-notebook` to get the same version in `all-spark-notebook`.
 
-- Spark distribution is defined by the combination of Spark, Hadoop, and Scala versions and verified by the package checksum,
-  see [Download Apache Spark](https://spark.apache.org/downloads.html) and the [archive repo](https://archive.apache.org/dist/spark/) for more information.
-
-  - `spark_version`: The Spark version to install (`3.3.0`).
-  - `hadoop_version`: The Hadoop version (`3.2`).
-  - `scala_version`: The Scala version (`2.13`, optional).
-  - `spark_checksum`: The package checksum (`BFE4540...`).
-  - `openjdk_version`: The version of the OpenJDK (JRE headless) distribution (`17`).
-    - This version needs to match the version supported by the Spark distribution used above.
-    - See [Spark Overview](https://spark.apache.org/docs/latest/#downloading) and [Ubuntu packages](https://packages.ubuntu.com/search?keywords=openjdk).
-
-- Starting with _Spark >= 3.2_, the distribution file might contain the Scala version.
-
-For example, here is how to build a `pyspark-notebook` image with Spark `3.2.0`, Hadoop `3.2`, and OpenJDK `11`.
-
-```{warning}
-This recipe is not tested and might be broken.
```

Comment:
This recipe doesn't work right now and the easiest thing is to remove it at all.
If someone wants to use old spark versions, we have tags.

And python script is not that difficult to change (or if someone wants to add customization but with proper test, I will gladly merge such a PR).